### PR TITLE
Bugfix/c1] methods should not be empty

### PR DESCRIPTION
**Pull request by nyneshreddy**
**What's in the PR**
An UnsupportedOperationException is thrown where empty methods are called.

**How to test manually**
* ...

**Automatic testing**
* [ ] PR includes unit tests

**Documentation**
* [ ] PR updates documentation

### CODE REVIEW

File: .sonarcloud.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (munterkalmsteiner) [inline .sonarcloud.properties]: Sonarcloud files should not end up in the repository

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+language: java
```
Comment (munterkalmsteiner) [inline .travis.yml]: Why is this change needed?

File: inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImplImportExportIntegrationTest.java [hunk 3]
```diff
@@ -103,6 +103,7 @@ public void tearDown() throws Exception {
 
     @Test
     public void thatApplicationContextStarts() {
+        throw new UnsupportedOperationException();
```
Comment (munterkalmsteiner) [inline inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImplImportExportIntegrationTest.java]: Since this is an empty test, wouldn't it be better to remove the test?

File: inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/pdfextract/PDFExtractor.java [hunk 4]
```diff
@@ -221,6 +221,7 @@ else if (obj instanceof DrawOperator) {
 
     @Override public void drawImage(PDImage pdImage)
     {
+        throw new UnsupportedOperationException();
```
Comment (munterkalmsteiner) [inline inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/pdfextract/PDFExtractor.java:224]: Hm, I see why you are throwing a runtime exception here, but I'm not sure that it is a good idea. It puts the program into an unrecoverable state since the exception cannot be caught. Maybe logging would be the less aggressive solution.

**Review by munterkalmsteiner — CHANGES_REQUESTED** [review]
There are a couple of issues with this PR (see comments).

File: inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImplImportExportIntegrationTest.java (hunk 3 above)
Comment (nyneshreddy) [reply inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImplImportExportIntegrationTest.java]: Removed the Empty test.

File: inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/pdfextract/PDFExtractor.java (hunk 4 above)
Comment (nyneshreddy) [reply inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/pdfextract/PDFExtractor.java:224]: logging could be a less aggressive option but in this case, the test is about the image. So, to my understanding, I think it would be better to throw an exception.

Comment (munterkalmsteiner) [reply inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/pdfextract/PDFExtractor.java:224]: ~~Which test? If the user opens a PDF with an image, then the pdfeditor fails horribly, instead of just not drawing the image.~~

File: inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImplIntegrationTest.java [hunk 5]
```diff
@@ -142,6 +142,7 @@ public void tearDown() throws Exception {
 
     @Test
     public void thatApplicationContextStarts() {
+        throw new UnsupportedOperationException();
```
Comment (munterkalmsteiner) [inline inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImplIntegrationTest.java:145]: Test should be deleted...

File: inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/pdfextract/PDFExtractor.java (hunk 4 above)
Comment (munterkalmsteiner) [reply inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/pdfextract/PDFExtractor.java:224]: Ok, you convinced me. Actually, I read [this](https://airbrake.io/blog/java-exception-handling/unsupportedoperationexception) and it makes sense to throw an UnsupportedOperationException here, as calling the method is a programming error and we want to know when such an error happens.

**Review by munterkalmsteiner — APPROVED** [review]
